Code/Framework/AzToolsFramework/AzToolsFramework/ActionManager/Menu/EditorMenu.cpp

```diff
@@ -198,7 +199,7 @@ namespace AzToolsFramework
                     }
                 case MenuItemType::SubMenu:
                     {
-                        if (QMenu* menu = s_menuManagerInternalInterface->GetMenu(menuItem.m_identifier))
+                        if (QMenu* menu = s_menuManagerInternalInterface->GetMenu(menuItem.m_identifier); !menu->isEmpty())
```

Comment:
question: The code before was checking to ensure `menu` wasn't null, but we're now we're not doing that anymore. Are we 100% sure that menu will never be `null` before we call `isEmpty` on it?

Reply:
Nice one thanks!